prober/utils.go

```diff
@@ -60,6 +61,7 @@ func chooseProtocol(ctx context.Context, IPProtocol string, fallbackIPProtocol b
 		fallbackProtocol = "ip6"
 	}
 
+	target = internationalizeDNSDomain(logger, target)
```

Comment:
below the `target` string is used is several places to log results.

Question: do I really want to see the actual target that was used (`www.xn--acadmie-franaise-npb1a.fr`) or do I want to see the target as entered by the user (`www.académie-française.fr`)? The first one is useful for debugging, the second one might be less surprising. Maybe this needs documenting?